src/pcedit/pc.editors.cpp

```diff
@@ -306,9 +306,10 @@ static bool pick_alchemy_led(cDialog& me, std::string item_hit, bool, bool allow
 		cLed& led = dynamic_cast<cLed&>(me.getControl(item_hit));
 		led.setState(univ.party.alchemy[hit] ? led_red : led_off);
 	}
-	// TODO: change message when hit==0
-	//       use a variable to store the beginning of the potion details
-	me["info"].setText(get_str("alchemy", hit+8));
+	if (hit==0)
+		me["info"].setText("Weak Curing\n  Min. Skill - 1\nNeed holly. Cures poison somewhat.");
```

Comment:
This is definitely wrong. The string shouldn't be hard-coded into the source code like this.

I think the correct answer is to `get_str` 8 and delete everything up to the first `|`. An alternate answer would be to split that string in two and add 9 instead of 8.

Reply:
Yes, it will be better to simply use strings 8...